sdk/search/Microsoft.Azure.Search/tests/SessionRecords/IndexingTests/CreateIndexerWithCacheReturnsCorrectDefinition.json

```diff
@@ -0,0 +1,714 @@
+{
```

Comment:
nit: It makes it easier to review if you make the session records changes in a separate commit.